template-processors/base/bin/resourceManager.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+# shellcheck disable=SC2002,SC2044,SC2154
```

Comment:
1. for SC2044, please don't disable it globally for whole file, but instead do explicit disable on each line where it is reported
2. for SC2154, similar as above - this one is very often a bug, so we definitively don't want to disable this for whole file